src/index.js

```diff
@@ -0,0 +1,49 @@
+import './style.css';
+import './asset/ellipsisverticalsolid.svg';
+
+const container = document.querySelector('.container');
+
+const tasks = [
+  {
+    description: 'Finish up the project',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Start new project',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Joggling',
+    completed: false,
+    index: 3,
+  },
+];
+
+const populate = () => {
+  tasks.forEach((task) => {
+    const wrapper = document.createElement('ul');
+    wrapper.classList.add('wrapper');
+
+    const li = document.createElement('li');
+    li.classList.add('list-item');
+    const horizontal = document.createElement('hr');
+    li.appendChild(horizontal);
+
+    const box = document.createElement('input');
+    box.setAttribute('type', 'checkbox');
+
+    const title = document.createElement('span');
+    title.classList.add('title');
+    title.innerHTML = task.description;
+    const option = document.createElement('i');
+    // option.innerHTML = '+++';
```

Comment:
### 🛑 Required Change
- [ ] Kindly delete commented out code. This helps in maintaining easy to read code

Reply:
@Cathella  I almost forgot that I have put those stuff when I was testing my stylesheet. 
Thanks :100: